synapse/storage/data_stores/main/__init__.py

```diff
@@ -483,36 +481,58 @@ def get_users(self):
         return self._simple_select_list(
             table="users",
             keyvalues={},
-            retcols=["name", "password_hash", "is_guest", "admin", "user_type"],
+            retcols=[
```

Comment:
Do we need to use `user_id` instead of `name` here as well?

Reply:
this refers to the name of the column in the table, so no, you'll have to stick with `name`.

I'd prefer it if the response from the API used `user_id` instead of `name`, but I'm not going to insist on that.